scripts/images/rancher-images.txt

```diff
@@ -17,11 +17,11 @@ docker.io/rancher/mirrored-library-busybox:1.32.1
 docker.io/rancher/mirrored-library-nginx:1.21.1-alpine
 docker.io/rancher/mirrored-prometheus-node-exporter:v1.1.2
 docker.io/rancher/mirrored-prometheus-node-exporter:v1.2.2
-docker.io/rancher/mirrored-prometheus-operator-prometheus-config-reloader:v0.48.0
-docker.io/rancher/mirrored-prometheus-operator-prometheus-operator:v0.48.0
-docker.io/rancher/mirrored-prometheus-prometheus:v2.27.1
-docker.io/rancher/rancher:v2.6.3-rc4
+docker.io/rancher/mirrored-prometheus-operator-prometheus-config-reloader:v0.50.0
+docker.io/rancher/mirrored-prometheus-operator-prometheus-operator:v0.50.0
+docker.io/rancher/mirrored-prometheus-prometheus:v2.28.1
 docker.io/rancher/rancher-webhook:v0.2.2-rc2
+docker.io/rancher/rancher:v2.6.3-rc5
```

Comment:
the latest rancher/rancher rc: v2.6.3-rc6

Reply:
thanks, actually because rc6 also has the same image issue so we hold the PR and waiting probably for the rc7.